lib/control-plane/openstackcontrolplane.yaml

```diff
@@ -182,3 +182,27 @@ spec:
         ringReplicas: 1
       swiftStorage:
         replicas: 1
+  telemetry:
```

Comment:
Do we want this in the lib/control-plane that ALL VAs/DTs use?

Perhaps it should be a file that this DT can access (and others) but not mandatory for all. I think this is similar Andrew's recent comment about `lib/control-plane/kustomization.yaml`

https://github.com/openstack-k8s-operators/architecture/pull/83#discussion_r1570943223

Reply:
Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

Reply:
marking as resolved for uni07 change.